src/embedding/embedding.jl

```diff
@@ -94,6 +115,11 @@ function berlekamp_massey(a::Array{Y, 1}, n::Int) where Y <: FieldElem
   return V1*lead(V1)^(-1)
 end
 
+
+#This function assumes that f has been created as a result of the embed_matrices
```

Comment:
Could we just remove the old version? Or is there something worth keeping that isn't implemented in the new version?